.expeditor/scripts/end_to_end/run_e2e_test_core.ps1

```diff
@@ -204,6 +204,15 @@ function Get-Leader($Remote, $ServiceGroup) {
     }
 }
 
+Function Invoke-BuildAndInstall($PackageName) {
+    hab pkg build test/fixtures/$PackageName
+    if($IsLinux) {
+        Get-Content "results/last_build.env" | ForEach-Object { Add-Content "results/last_build.ps1" -Value "`$$($_.Replace("=", '="'))`"" }
```

Comment:
This is converting `.env` files to `.ps1` files, correct? I am sure it is correct, but I am having a hard time following what the replace is doing. Maybe I am just not used to that syntax, but it might be worth adding a comment.

Reply:
I'll add a comment to explain but its basically changing the format from `var=value` to `$var='value'` that can be parsed by powershell.